crypto/cmake/crypto-config.cmake

```diff
@@ -2,11 +2,22 @@ if(WIN32 OR UNIX OR APPLE)
     find_package(Threads REQUIRED)
 endif()
 
+# Allow static or shared lib to be used.
+# If both are installed, choose based on BUILD_SHARED_LIBS.
 if (BUILD_SHARED_LIBS)
-    message(STATUS "FOUND AWS-LC CRYPTO cmake config - shared")
-    include(${CMAKE_CURRENT_LIST_DIR}/shared/crypto-targets.cmake)
+    if (EXISTS "${CMAKE_CURRENT_LIST_DIR}/shared/crypto-targets.cmake")
+        message(STATUS "FOUND AWS-LC CRYPTO cmake config - shared")
+        include(${CMAKE_CURRENT_LIST_DIR}/shared/crypto-targets.cmake)
+    else()
+        message(STATUS "FOUND AWS-LC CRYPTO cmake config - static")
```

Comment:
This was present in existing code, but is this message accurate? We don't actually know yet...

Reply:
Good point!
I moved the `message()` after the `include()`